mods/species/bayliens/skrell/_skrell.dm

```diff
@@ -1,5 +1,4 @@
 #define SPECIES_SKRELL "Skrell"
-#define BODYTYPE_SKRELL "skrellian body"
```

Comment:
Why remove this?

Reply:
But you're using the same value, just plain string now instead of define, what's different?